functions/source/ConfigParser/params.py

```diff
@@ -0,0 +1,106 @@
+import urllib.parse
+from typing import Dict
+from dataclasses import dataclass
+import json
+import re
+import boto3
+
+########################
+##  Helper Functions  ##
+########################
+
+# Input is a string that may or may not be URL encoded
+# This function returns the string, as-is, if not URL encoded
+# Returns decoded if is URL encoded
+def safe_decode(string):
+    try_decode = urllib.parse.unquote(string)
+    if len(string) > len(try_decode):
+        return try_decode
+    return string
+
+def is_http(string):
+    pattern = r"^http(s):\/\/.*"
+    is_match = bool(re.match(pattern,string))
+    return is_match
+
+def is_s3(string):
+    pattern = r"^s3:\/\/.*"
+    is_match = bool(re.match(pattern,string))
+    return is_match
+
+def get_from_http(url):
+    f = requests.get(url)
+    return f.text
+
+def get_from_s3(uri):
+    bucket = uri.split("//")[1].split('/')[0]
+    key_list = uri.split("//")[1].split('/')[1:]
+    file_key = "/".join(key_list)
```

Comment:
split once?
```suggestion
    bucket, file_key = uri.split("//")[1].split("/", maxsplit=1)
```

Reply:
Super nifty! That's some ninja-level string manipulation 🐱‍👤